### ISSUE TITLE
dwarf expansion

### ISSUE THREAD

**Issue by antio789** [antio789/warhammermod#2]
dwarf villages in mountains with unique trades and new weapons like flamethrower and torpedo launcher

**Comment by MokoBlack** [comment]
Maybe can let  dwarf become  fighting villager ,you know .. in fantasy game or novel dwarf always  good as combat

**Comment by antio789** [comment]
> Maybe can let dwarf become fighting villager ,you know .. in fantasy game or novel dwarf always good as combat

yes absolutely is planed, with a grudge mecanism they attack as response (like the zombiepigman basically)

**Comment by MokoBlack** [comment]
By the way, is your mod based on the Warhammer-Fantasy?
tbh , Your mod makes me interested in the Warhammer game

**Comment by antio789** [comment]
yes it is, I fell in love with this universe after playing total war warhammer. 
But if you are interessted there are plenty of them (another one I use to design the weapons is vermintide)

**Comment by MokoBlack** [comment]
Yes, what I'm most interested in is the theme design of fantasy technology. I don't like the feeling of high technology very much. This kind of medieval retro science fiction is very popular with me.
For MOD, I'm also focused on medieval themes.
